### DISCUSSION TITLE
Mix text in formula field

### DISCUSSION THREAD

**Discussion by InanisOmnia** [mdelobelle/metadatamenu#588] [Tips and Tricks]
I need to be able to write text in with my formula
For example I have something that calculates the amount of pay I am going to get from a certain number of working hours and I would like to prepend the "£" sign
Is there a way to mix literal strings with the calculation

e.g. database folder does this, where anything inside ${ ... } will be executed, like JS backtick strings
```
This is literal text £${current.hours * current.myRate}
```

**Comment by fabricetayou** [comment]
Maybe you meant 
```JavaScript
"This is literal text " + (current.hours * current.myRate}`
```
This should produce the same output

**Comment by mdelobelle** [comment]
Yes with string formatting: 
```
`This is literal text ${current.hours * current.myRate}`
```
